core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/datastructures/ViewNames.java

```diff
@@ -176,6 +178,7 @@ public static ViewTypes getViewType(String viewName) {
             case COREF_EXTENT:
             case COREF_ERE:
                 return ViewTypes.COREF_VIEW;
+            case TIMEX3:
```

Comment:
What type of view will TIMEX3 be? Right now, it is of type NULL. Move the case to the block that contains the correct view type.